README.md

```diff
@@ -93,6 +93,11 @@ via an app password as suspicious. Should it detect a password login where the (
 uid) is classified as suspicious by the trained model, it will add an entry to the
 ``suspicious_login`` table, including the timestamp, request id and URL.
 
+## Configuration
+
+Enable a button in notifications to lookup more info about a suspicious IP (geography, ISP):
+
+`occ config:app:set --value 1 suspicious_login show_more_info_button`
```

Comment:
This is the default iirc so rather document how to disable the feature?

Reply:
Right, it is enabled by default. @joshtrichards Could you please invert the README in another PR?

https://github.com/nextcloud/suspicious_login/blob/30b748f6f3469026914888fa552f9f44004de7df/lib/Notifications/Notifier.php#L89

Reply:
Good point. #876